packages/cli-kit/src/plugins.ts

```diff
@@ -5,12 +5,18 @@ import {PickByPrefix} from './typing/pick-by-prefix.js'
 import {MonorailEventPublic} from './monorail.js'
 import {Interfaces} from '@oclif/core'
 
+export type TunnelHook = `tunnel_start_${string}`
```

Comment:
This `plugins.ts` is a good candidate to grow into a huge module. As soon as we see this one growing too much we should consider breaking it down into smaller and more manageable modules.